# from app.utils.helpers import validate_outage_data
#
# def test_validate_outage_data_valid():
#     valid_data = {"location": "Tbilisi", "outage_start": "2024-11-12T12:00:00"}
#     result = validate_outage_data(valid_data)
#     assert result is True
#
# def test_validate_outage_data_invalid():
#     invalid_data = {"location": "", "outage_start": ""}
#     result = validate_outage_data(invalid_data)
#     assert result is False
